packages/cli/src/cms/commands/push-status.ts

```diff
@@ -67,7 +68,7 @@ export async function handlePushStatus({
 
   try {
     const apiKey = getApiKeys(domain);
-    const client = new ReuniteApiClient(domain, apiKey);
+    const client = new ReuniteApiClient(domain, apiKey, version, 'push-status');
```

Comment:
Can we use it this way for better readability?

```suggestion
    const client = new ReuniteApiClient({domain, apiKey, version, command: 'push-status'});
```